Dockerfile.rhel

```diff
@@ -1,11 +1,10 @@
-FROM openshift/origin-release:golang-1.10 as build
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.10 AS builder
```

Comment:
Please look exactly like this commit (
https://github.com/openshift/csi-external-attacher/pull/6).  You want two docker files, one with `.rhel` as an extension.

Reply:
Done